redux/js/components/ProductsContainer.jsx

```diff
@@ -18,26 +17,23 @@ class ProductItemContainer {
             <ProductItem
                 product={product}
                 {...bindActionCreators({
-                    // es7 bind syntax (https://github.com/zenparsing/es-function-bind)
-                    onAddToCartClicked: ::this.onAddToCartClicked
+                    onAddToCartClicked: this.onAddToCartClicked.bind(this)
                 }, dispatch)}
             />
         );
     }
 }
 
-@connect(state => ({
-    // named in stores/index.js
-    products: state.products
-}))
-export default class ProductsListContainer {
+const ConnectedProductItemContainer = connect(() => ({}))(ProductItemContainer);
```

Comment:
We changed to Babel `stage: 2` so we have to manually apply decorators. But I'll check if I can do it better :P